README.md

```diff
@@ -37,10 +37,31 @@ Wanda is exposed on port `4000` and the API documentation is available at http:/
 
 With a running setup, it is possible to easily test Checks and their Execution by:
 
+- starting the targets
 - consulting the catalog
 - starting a Checks Execution
 - checking the state of an execution
 
+#### **Starting the targets**
+
+The [trento-agent](https://github.com/trento-project/agent) must be up and running on the targets to run a correct execution, otherwise a timeout error is raised.
+
+Here an example on how to start it:
+
+```
+./trento-agent start --ssh-address 0 --api-key 0 --factsengine --facts-service-url amqp://wanda:wanda@localhost:5674
```

Comment:
What about using some placeholders instead of `0`

`./trento-agent start --ssh-address <an-ssh-address> --api-key <an-api-key> --factsengine --facts-service-url...`

I understand that those options are not really needed for the scenario of checks execution, however `0` might result a bit confusing, at least to me :sweat_smile:

Reply:
You are right!
I put placeholders and note comment to be more helpful